package.json

```diff
@@ -41,62 +55,37 @@
 	},
 	"homepage": "https://github.com/adonisjs/view#readme",
 	"devDependencies": {
-		"@adonisjs/core": "^5.9.0",
-		"@adonisjs/mrm-preset": "^5.0.3",
-		"@adonisjs/repl": "^3.1.11",
-		"@adonisjs/require-ts": "^2.0.13",
-		"@adonisjs/sink": "^5.4.2",
-		"@japa/assert": "^1.3.6",
-		"@japa/run-failed-tests": "^1.1.0",
-		"@japa/runner": "^2.2.2",
-		"@japa/spec-reporter": "^1.3.2",
-		"@poppinss/dev-utils": "^2.0.3",
-		"@types/node": "^18.11.9",
-		"commitizen": "^4.2.5",
-		"copyfiles": "^2.4.1",
-		"cz-conventional-changelog": "^3.3.0",
+		"@adonisjs/core": "^6.1.5-2",
+		"@commitlint/cli": "^17.6.1",
+		"@commitlint/config-conventional": "^17.6.1",
+		"@japa/assert": "^1.4.1",
+		"@japa/expect-type": "^1.0.3",
+		"@japa/file-system": "^1.0.1",
+		"@japa/run-failed-tests": "^1.1.1",
+		"@japa/runner": "^2.5.1",
+		"@japa/spec-reporter": "^1.3.3",
+		"@swc/core": "^1.3.51",
+		"@types/node": "^18.15.11",
+		"c8": "^7.13.0",
 		"del-cli": "^5.0.0",
-		"eslint": "^8.28.0",
-		"eslint-config-prettier": "^8.5.0",
+		"eslint": "^8.38.0",
+		"eslint-config-prettier": "^8.8.0",
 		"eslint-plugin-adonis": "^2.1.1",
 		"eslint-plugin-prettier": "^4.2.1",
-		"github-label-sync": "^2.2.0",
-		"husky": "^8.0.2",
-		"japa": "^4.0.0",
-		"mrm": "^4.1.13",
-		"np": "^7.6.2",
-		"prettier": "^2.8.0",
-		"typescript": "^4.8.2"
+		"github-label-sync": "^2.3.1",
+		"husky": "^8.0.3",
+		"np": "^7.7.0",
+		"prettier": "^2.8.7",
+		"ts-node": "^10.9.1",
+		"typescript": "^5.0.4"
 	},
 	"peerDependencies": {
-		"@adonisjs/core": "^5.7.0"
-	},
-	"nyc": {
-		"exclude": [
-			"test"
-		],
-		"extension": [
-			".ts"
-		]
-	},
-	"husky": {
-		"hooks": {
-			"commit-msg": "node ./node_modules/@adonisjs/mrm-preset/validateCommit/conventional/validate.js"
-		}
-	},
-	"config": {
-		"commitizen": {
-			"path": "cz-conventional-changelog"
-		}
-	},
-	"np": {
-		"contents": ".",
-		"anyBranch": false
+		"@adonisjs/core": "^6.0.0"
```

Comment:
This might fail the semver check. So we will have to use `^6.1.5-2`. I use this app to see which expression matches which versions. https://semver.npmjs.com/